### PR TITLE
Form error handling

### PR DESCRIPTION
**Pull request by Leoy23**
# Pull Request
Add Form Error Handling

## Category: [Fix] [Feature] [Suggestion] [Issue] [Style] [Docs] [Chore]
feature/fix

## Description:
- Add error handling to editCocktail form
- Update/fix error handling for search form
- condense search query

## Next Steps:
- Complete mutation query for adding a drink to database 
- Create structure for Profile Page???

## Any other comments or questions:
No ma'am

### CODE REVIEW

File: src/EditCocktail/EditCocktail.js [hunk 1]
```diff
@@ -92,8 +98,13 @@ const EditCocktail = ({ choosenCocktail, updateCocktail, updateSteps }) => {
   };
 
   const submitEdit = () => {
-    console.log(allIngredients);
-    console.log(steps);
+    if (allIngredients.length === 0 || steps.length === 0) {
+      setError(true)
+      setMessage("Please fill out all fields!");
+    } else {
+      setMessage("Saved Successfully!")
```
Comment (piperjarrett) [inline src/EditCocktail/EditCocktail.js:105]: I like how you do a happy path when they submit!

File: src/EditCocktail/EditCocktail.js [hunk 2]
```diff
@@ -92,8 +98,13 @@ const EditCocktail = ({ choosenCocktail, updateCocktail, updateSteps }) => {
   };
 
   const submitEdit = () => {
-    console.log(allIngredients);
-    console.log(steps);
+    if (allIngredients.length === 0 || steps.length === 0) {
```
Comment (piperjarrett) [inline src/EditCocktail/EditCocktail.js:101]: We also need to add the cocktailName and check it!

**Review by piperjarrett — COMMENTED** [review]
Good Job! We just need to add error handling for the name

File: src/EditCocktail/EditCocktail.js (hunk 2 above)
Comment (Leoy23) [reply src/EditCocktail/EditCocktail.js:101]: Cool, I will do that now

**Review by mmartinelli22 — COMMENTED** [review]
Sorry for the late response, looking good! Thanks for cleaning up the console logs and stuff too.

**Pull request merged**